backend/src/models/db_lock/index.js

```diff
@@ -32,7 +32,7 @@ const getLockByParams = (requestData, reply) => {
       if (0 === rows?.length) {
         //No content (204)
         //TODO Move the reply functionality to the plugin backend/src/plugins/fastifyRowLockCheck/index.js
-        reply.code(204);
+        // reply.code(204);
```

Comment:
I think we commented this out, but not sure if it should be.  just flagging this, if it is still ok to leave commented out, then im fine due to the todo that is in place

Reply:
I'm going to fix this coming up